app/src/main/java/com/example/campusguide/MapsActivity.kt

```diff
@@ -93,9 +93,14 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
         // Update switch campus button listener
         val switchCampusToggle: ToggleButton = findViewById(R.id.switchCampusButton)
         SwitchCampus(switchCampusToggle, mMap)
-    }
 
+        mMap.setContentDescription("Google Maps Ready")
+    }
 
+    override fun onDestroy() {
+        super.onDestroy()
+        ObjectBox.boxStore.close()
+    }
```

Comment:
@FGRCL 
Ever since your database+highlights changes got merged, the first test in a test class passes but the second consistently fails with the error "Another BoxStore is still open for this directory"
I decided to close the boxStore right before the activity is destroyed
Let me know if this makes sense, I might have overlooked something

Reply:
From the documentation, this method is mostly intended for unit tests
[https://objectbox.io/files/objectbox-java/current/io/objectbox/BoxStore.html#close()]
)So I'd put it in a @After in the tests classes like this
`@After
    fun endTest(){
        ObjectBox.boxStore.close()
}`

Reply:
That's a much better solution